docs/workflows/transitions/preprocess/preprocess/utils.py

```diff
@@ -114,3 +121,77 @@ def merge_predictions_and_gt(predictions, old_ground_truth, field_config):
 
     return updated_predictions
 
+
+def overlap(last_lines, first_lines):
+    for first_line in first_lines:
+        for last_line in last_lines:
+            if first_line['label'] == last_line['label'] and first_line['value'] != last_line['value']:
+                return False
+    return True
+
+
+def merge_lines(last_line, first_line):
+    return last_line + first_line
+
+
+def merge_lines_from_different_pages(predictions, field_config):
+    line_labels = [field for field, config in field_config.items() if config['type'] == 'lines']
+
+    if not line_labels:
+        return predictions
+
+    line_predictions = {line_field: [] for line_field in line_labels}
+
+    merged_predictions = []
+    for prediction in predictions:
+        label = prediction['label']
+        if label in line_labels:
+            line_values = prediction['value']
+
+            # Check lines within the prediction (can have up to 3 pages within a prediction)
+            previous_line = line_values[0]
```

Comment:
Is `line_values` always nonempty?

Reply:
Not sure.. I will make a check for it just in case!